stripe-ruby-mock.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.require_paths = ['lib']
 
-  gem.add_dependency 'stripe', '>=1.31.0'
+  gem.add_dependency 'stripe', '>=1.40.0'
```

Comment:
I don't think that we can do that now. A lot of users still use old version of stripe. We can update it later